client/base/src/main/java/io/a2a/client/config/ClientConfig.java

```diff
@@ -12,21 +12,18 @@ public class ClientConfig {
 
     private final Boolean streaming;
     private final Boolean polling;
-    private final List<ClientTransportConfig> clientTransportConfigs;
     private final List<String> supportedTransports;
     private final Boolean useClientPreference;
     private final List<String> acceptedOutputModes;
     private final PushNotificationConfig pushNotificationConfig;
     private final Integer historyLength;
     private final Map<String, Object> metadata;
 
-    public ClientConfig(Boolean streaming, Boolean polling, List<ClientTransportConfig> clientTransportConfigs,
-                        List<String> supportedTransports, Boolean useClientPreference,
+    public ClientConfig(Boolean streaming, Boolean polling, List<String> supportedTransports, Boolean useClientPreference,
                         List<String> acceptedOutputModes, PushNotificationConfig pushNotificationConfig,
                         Integer historyLength, Map<String, Object> metadata) {
         this.streaming = streaming == null ? true : streaming;
         this.polling = polling == null ? false : polling;
-        this.clientTransportConfigs = clientTransportConfigs;
         this.supportedTransports = supportedTransports;
```

Comment:
Was just looking closer at `ClientConfig` after Emmanuel's question and just realized that we should remove `supportedTransports` from here since that's now determined by the the transports that are configured with `withTransport`. WDYT?